composer.json

```diff
@@ -35,7 +35,7 @@
         "doctrine/annotations": "^1.2 || ^2.0",
         "doctrine/cache": "^1.11 || ^2.0",
         "doctrine/collections": "^1.0 || ^2.0",
-        "doctrine/data-fixtures": "^1.3.3",
+        "doctrine/data-fixtures": "^1.3.3 || ^2.0",
         "doctrine/dbal": "^2.13 || ^3.0",
         "doctrine/doctrine-bundle": "^2.4",
         "doctrine/doctrine-fixtures-bundle": "^3.3 || ^4.0",
```

Comment:
actually doctrine-fixtures-bundle 4 requires data-fixtures 2 and was currently not installed.